src/BenchmarkDotNet/Validators/DotNetSdkVersionValidator.cs

```diff
@@ -0,0 +1,102 @@
+﻿using BenchmarkDotNet.Jobs;
+using BenchmarkDotNet.Toolchains;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+
+namespace BenchmarkDotNet.Validators
+{
+    public class DotNetSdkVersionValidator : IValidator
```

Comment:
I'm not sure if this needs to be a public `IValidator`. I think it could just be a static utility class that the toolchains call. If you want public validators, do separate ones for Core and for Framework. But I don't think it's necessary since the toolchains automatically do the validation.